packages/vaadin-notification/test/styling.test.js

```diff
@@ -78,30 +79,10 @@ describe('notification content styling', () => {
       delete notification.constructor._container;
     });
 
-    it('should stamp content into a shadowRoot', () => {
```

Comment:
I would remove this test file altogether because the related code has been removed.

We can preserve the test on the line 116 but move it to the different file.

```
it('should stamp content into notification overlay host', () => {
```

Reply:
Should I preserve the following test as well?

```js
describe('theme attribute', () => {
  let notification;

  beforeEach(() => {
    notification = fixtureSync(`
      <vaadin-notification theme="foo" duration="0">
        <template></template>
      </vaadin-notification>
    `);
  });

  it('should propagate theme attribute to card', () => {
    const card = notification.shadowRoot.querySelector('vaadin-notification-card');
    expect(card.getAttribute('theme')).to.equal('foo');
  });
});
```

Reply:
Decided to preserve this test, and now it is at `notification.test.js`.